Quotient/room.h

```diff
@@ -838,7 +838,7 @@ public Q_SLOTS:
     /// The event is about to be appended to the list of pending events
     void pendingEventAboutToAdd(Quotient::RoomEvent* event);
     /// An event has been appended to the list of pending events
-    void pendingEventAdded();
+    void pendingEventAdded(Quotient::RoomEvent* event);
```

Comment:
```suggestion
    void pendingEventAdded(const Quotient::RoomEvent* event);
```

Reply:
I'll go ahead and commit it because that's what the pending item holds, anyway - passing the event non-const here would give an unwanted backdoor.